### PR TITLE
Updates and new features

### PR DESCRIPTION
**Pull request by lparis**
@mcqueary @ColinSullivan1 @kozlovic

### CODE REVIEW

File: content/documentation/server/gnatsd-config.md [hunk 1]
```diff
@@ -40,9 +42,15 @@ The config file format supports the following syntax:
...
+In general the configuration parameters are the same as the [command line arguments](http://nats.io/documentation/server/gnatsd-usage/). Note, however, the following differences:
+
+- The listen option is host:port for connections, on the server cli its -a and -p no hostport is supported.
+- http/https is only port on the cli, on the config it is host:port (there’s no config flag for the interface for the monitoring)
+- The -cluster flag is used for defining the host:port where routes can be solicited, on the config file this is called ‘listen’ as part property of a ‘cluster’ object.
+
 ## Sample server config file
 
-The following demonstrates an examples NATS server config file:
+The following demonstrates an example NATS server config file. See also the [NATS Server README]
```
Comment (kozlovic) [inline content/documentation/server/gnatsd-config.md]: Not sure what of the above is causing the problem, but the following sample configuration file is not appearing in a code section (as seen with hugo).

**Comment by kozlovic** [comment]
LGTM. For some reason, when looking at the site with hugo, I am getting the config files showing as plain text (```not like this```).

**Comment by mcqueary** [comment]
LGTM

**Comment by ColinSullivan1** [comment]
LGTM.

**Comment by lparis** [comment]
@chriscasper @mcqueary I see that the nats-server version 0.9.0.beta is in the nats-streaming-server, so this can be merged. But, I don't want to mess up the site by doing so, so I think maybe @chriscasper you should do it. Thanks.

**Comment by mcqueary** [comment]
LGTM and agree @chriscasper should merge stuff for this.

On Sat, Jun 25, 2016 at 10:06 AM, Lorenzo Paris <notifications@github.com>
wrote:

> @chriscasper <https://github.com/chriscasper> @mcqueary
> <https://github.com/mcqueary> I see that the nats-server version
> 0.9.0.beta is in the nats-streaming-server, so this can be merged. But, I
> don't want to mess up the site by doing so, so I think maybe @chriscasper
> <https://github.com/chriscasper> you should do it. Thanks.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/nats-io/nats-site/pull/111#issuecomment-228552725>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGq4LNyf5a8kTEU-lsMbl0AV-wqy2ruPks5qPVHvgaJpZM4I9ZJe>
> .
>



-- 
*Larry McQueary* | Director, Messaging Technology
larry@apcera.com | @mcqueary | github.com/mcqueary

**Comment by chriscasper** [comment]
Will take care of this.

**Comment by chriscasper** [comment]
Merge conflict taken care of and merged.

**Pull request merged**